torchrec/distributed/train_pipeline.py

```diff
@@ -81,10 +82,12 @@ def __init__(
         model: torch.nn.Module,
         optimizer: torch.optim.Optimizer,
         device: torch.device,
+        dataloader_iter: Iterator[In] = None,
```

Comment:
Optional[Iterator[In]]

A couple of lint errors - I think 
```
pip install pre-commit
pre-commit
```

can catch them 

@divchenko